plugins/net_plugin/net_plugin.cpp

```diff
@@ -3927,16 +3975,24 @@ namespace eosio {
    void net_plugin::set_program_options( options_description& /*cli*/, options_description& cfg )
    {
       cfg.add_options()
-         ( "p2p-listen-endpoint", bpo::value< vector<string> >()->default_value( vector<string>(1, string("0.0.0.0:9876")) ), "The actual host:port used to listen for incoming p2p connections. May be used multiple times.")
+         ( "p2p-listen-endpoint", bpo::value< vector<string> >()->default_value( vector<string>(1, string("0.0.0.0:9876:0")) ), "The actual host:port used to listen for incoming p2p connections. May be used multiple times. "
+           "Block syncing to all peers connected via the port will be throttled to the specified rate. "
```

Comment:
Need some examples of valid configurations.

Reply:
I put them all on the wrong option.  Moved from `p2p-peer-address`.